users/src/commonMain/kotlin/majestic/users/profile/roles/assignRoleModalContent/AssignRoleHeader.kt

```diff
@@ -0,0 +1,64 @@
+package majestic.users.profile.roles.assignRoleModalContent
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import majestic.users.labels.roles.RolesLabels
+import majestic.users.profile.roles.RolesColors
+import org.jetbrains.compose.resources.painterResource
+import tz.co.asoft.majestic_users.generated.resources.Res
+import tz.co.asoft.majestic_users.generated.resources.ic_account_settings_filled
+
+@Composable
+fun AssignRoleHeader(
+    userName: String,
+    labels: RolesLabels.AssignRoleModalLabels,
+    colors: RolesColors
+) {
+    Row(
+        horizontalArrangement = Arrangement.SpaceBetween,
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = Modifier
+            .fillMaxWidth()
+            .background(Color(0xFF202733))
+            .padding(20.dp),
```

Comment:
always expose your composables. Do not hard code them inside a particual composable. this makes it easy to see how it is used on the call site and it minimizes confusion